enhancements/machine-api/control-plane-machine-set.md

```diff
@@ -983,7 +1019,10 @@ TBD
 
 #### Tech Preview -> GA
 
-TBD
+##### MachineNamePrefix Functionality
+- Removal (enabled by default) of `CPMSMachineNamePrefix` feature gate usage.
```

Comment:
Nit, `Removal` is not the right word here, prefer to use `Promote` the feature gate

Reply:
Thank you, updated